### ISSUE TITLE
Do not rely on localeapp flags

### ISSUE THREAD

**Issue by alepore** [spree-contrib/spree_globalize#18]
We use images from `https://www.localeapp.com/assets/flags/languages/<%= locale %>.png` and similar.
This works well but will break al the apps of the world when localeapp will decide to change something.
We don't even use localeapp :)

**Comment by tvdeyen** [comment]
Depart from any potential copyright violations. ;)

Adding a free flag set like http://www.famfamfam.com/lab/icons/flags/ would be awesome.

**Comment by alepore** [comment]
One of the problems i remember is that there's not full locale => country code match in "normal" country flags sets

**Comment by tvdeyen** [comment]
Yes. That's the problem with flags for languages. Language != Country.

Why use language flags anyway? Why not just display the locale and maybe a humanized name? We used to have language flags in AlchemyCMS but removed them and use locale names instead. 

Then we can spare us serving flags completely and won't offense people with wrong flags representing their language.

**Comment by alepore** [comment]
:+1: :+1: :+1:

**Comment by alepore** [comment]
https://twitter.com/localeapp/status/644082693878235136

**Comment by tvdeyen** [comment]
Nice! But, dependencies (`librsvg`) just for flags? Don't know. 

I still vote removing the flags altogether.

**Comment by alepore** [comment]
i think that's just to generate the files the first time (but haven't checked yet)
